node/parachain/src/chain_spec.rs

```diff
@@ -49,11 +68,16 @@ pub fn get_chain_spec(para_id: ParaId) -> ChainSpec {
 	ChainSpec::from_genesis(
 		"Moonbase Parachain Local Testnet",
 		"local_testnet",
-		ChainType::Local,
+		ChainType::Development,
```

Comment:
What does it impact ?

Reply:
My previous understanding was that this would give Apps a clue to include Alice, Bob, etc in the interface. It did not work, so I need to dig more into it.